nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/usage.md

```diff
@@ -229,9 +225,12 @@ Specify the path to a specific config file (this is a core NextFlow command).
 
 Note - you can use this to override pipeline defaults.
 
+### `--custom_config_version`
+Provide git commit id for custom Institutional configs hosted at `nf-core/configs`. This was implemented for reproducibility purposes. Default is set to `master`.
```

Comment:
Maybe give an example of providing an exact commit ID..

Reply:
Added in latest commit.